upstream-builder.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.16-alpine as builder
+FROM quay.io/fedora/fedora:34-x86_64 as builder
```

Comment:
Was there a reason to change this from `alpine`? 

I suspect this image build is also broken now due to the `apk` calls below.